src/main/java/com/softserve/controller/StudentController.java

```diff
@@ -27,10 +25,10 @@
 public class StudentController {
 
     private final StudentService studentService;
-    private final StudentMapper studentMapper;
+    private final StudentMapperNew studentMapper;
```

Comment:
StudentMapperNew - is not the correct name. Please rename

Reply:
Done